src/main/java/Todo.java

```diff
@@ -0,0 +1,19 @@
+//Matric Number: A0200273X
+public class Todo extends Task {
+
+    public Todo(String description) {
+        super(description);
+    }
+
+    //Override Methods when to do class is called
+    @Override
```

Comment:
Is there a reason to override when the both Task and Todo is similar for this function?